client/js/lounge.js

```diff
@@ -111,6 +111,64 @@ $(function() {
 		index: 1
 	};
 
+	const colorCodeMap = [
+		[0, "White"],
+		[1, "Black"],
+		[2, "Blue"],
+		[3, "Green"],
+		[4, "Red"],
+		[5, "Brown"],
+		[6, "Magenta"],
+		[7, "Orange"],
+		[8, "Yellow"],
+		[9, "Light Green"],
+		[10, "Cyan"],
+		[11, "Light Cyan"],
+		[12, "Light Blue"],
+		[13, "Pink"],
+		[14, "Grey"],
+		[15, "Light Grey"],
+	];
+
+	const foregroundColorStrategy = {
+		id: "foreground-colors",
+		match: /\B\x03(\d{0,2}|[A-Za-z ]{0,10})$/,
```

Comment:
Ok, it looks like it's working now. I probably messed something with my regexes earlier, nevermind...